src/hooks/index.js

```diff
@@ -327,6 +327,24 @@ const testIfRoleNameExist = (user, roleNames) => {
 	return user.roles.some(({ name }) => roleNames.includes(name));
 };
 
+exports.enableQuery = (context) => {
+	if (context.id) {
+		context.params.query._id = context.id;
+		context.id = null;
+	}
+};
+
+exports.enableQueryAfter = (context) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected to return a value at the end of arrow function.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5387400)